### PR TITLE
magento/devdocs#: Get Redis version on Cloud environments

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information how a Redis version may be retrieved on Cloud instances.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/project/services-redis.html

## Links to Magento source code

redis

Thank you!

### CODE REVIEW

File: src/cloud/project/services-redis.md [hunk 1]
```diff
@@ -63,3 +63,26 @@ Assuming your Redis relationship is named `redis`, you can access it using the `
...
+```bash
+redis-cli -h redis.internal info | grep version
+```
+
+```terminal
+redis_version:5.0.0
+gcc_version:6.3.0
+```
+
+To get a Redis version on the Staging and Production environments use `redis-server` command:
```
Comment (meker12) [inline src/cloud/project/services-redis.md:80]:
```suggestion
To get the Redis version installed in a Staging or Production environment, use the `redis-server` command:
```

File: src/cloud/project/services-redis.md [hunk 2]
```diff
@@ -63,3 +63,26 @@ Assuming your Redis relationship is named `redis`, you can access it using the `
    ```bash
    redis-cli -h redis.internal
    ```
+
+## Get installed Redis version
+
+Use the following command to get a Redis version on the Integration environment:
```
Comment (meker12) [inline src/cloud/project/services-redis.md:69]:
```suggestion
Use the following command to get the Redis version installed in an Integration environment:
```

**Review by meker12 — CHANGES_REQUESTED** [review]

File: src/cloud/project/services-redis.md (hunk 2 above)
Comment (atwixfirster) [reply src/cloud/project/services-redis.md]: **done**

File: src/cloud/project/services-redis.md (hunk 1 above)
Comment (atwixfirster) [reply src/cloud/project/services-redis.md]: **done**

File: src/cloud/project/services-redis.md (hunk 2 above)
Comment (atwixfirster) [reply src/cloud/project/services-redis.md]: Thank you, @meker12

Comment (BarnyShergold) [reply src/cloud/project/services-redis.md]: Normally we'd say "ON an Integration environment"

Comment (BarnyShergold) [reply src/cloud/project/services-redis.md]: Same goes for above on the Staging & Production

Comment (atwixfirster) [reply src/cloud/project/services-redis.md]: **done**

@BarnyShergold , could you please review?

Thank you!

**Review by BarnyShergold — APPROVED** [review]
Sorry to be super picky but you know me.....

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
Thanks for the great addition @atwixfirster :-)

**Pull request merged**